bitex/formatters/base.py

```diff
@@ -19,8 +19,8 @@ class APIResponse(requests.Response, metaclass=ABCMeta):
     def __init__(self, method, response_obj, *args, **kwargs):
         """Initialize the object."""
         if not isinstance(response_obj, requests.Response):
-            raise TypeError("Response obj must be requests.Response instance, "
-                            "not %s" % type(response_obj))
+            raise TypeError("Response obj must be requests.Response instance, not %s"
+                            % type(response_obj))
```

Comment:
Be a better developer than me and convert this to
```
raise TypeError("Response obj must be requests.Response instance, not {}".format(type(response_obj)))
```

Reply:
all % change to "{}".format